### PR TITLE
Add coverity in travis

### PR DESCRIPTION
**Pull request by jvoisin**
This commit makes travis automatically upload data to coverity.

Since the email address is mandatory, I put a useless one there. I tested this on my fork, it's working™

### CODE REVIEW

**Comment by jooola** [comment]
What about using the airsonic email ? @muff1nman made one for airsonic on tutanova. Check the password repo at airsonic/passwords

**Comment by jvoisin** [comment]
I don't have access to the passwords in clear-text :D 
But sure, we can change the email address to whatever you want ♥

**Comment by muff1nman** [comment]
Yes lets make sure that any accounts/emails are under the airsonic-passwords umbrella. The main airsonic email is `airsonic@tutanota.com`

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,22 @@ script:
...
+        - echo "coverity scan install"
+      addons:
+        coverity_scan:
+          project:
+            name: "airsonic/airsonic"
+            description: "A Free and Open Source community driven media server"
+          notification_email: "airsonic@tutanota.com"
+          build_command_prepend: mvn clean
+          build_command: mvn -DskipTests=true package
+          branch_pattern: master
```
Comment (muff1nman) [inline .travis.yml]: Same as #1051 - can we do this in maven instead?

Comment (jvoisin) [reply .travis.yml]: I would be happy to, but I don't know Maven at all :/

**Comment by jvoisin** [comment]
Can we get this merged? As [I said]( https://github.com/airsonic/airsonic/pull/1051#discussion_r303202599 ):

- the .travis-ci.yml and .gitlab-ci.yml are simple enough
- everyone knows yaml, fewer know maven lifecycles
- having this in the yaml files is the recommended way to use coverity

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Airsonic
 ========
 [![Build Status](https://travis-ci.org/airsonic/airsonic.svg?branch=master)](https://travis-ci.org/airsonic/airsonic)
-[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic)
+[![Coverity scan](https://scan.coverity.com/projects/18356/badge.svg)](https://scan.coverity.com/projects/airsonic-airsonic)
```
Comment (muff1nman) [inline README.md]: why the change in url?

Comment (jvoisin) [reply README.md]: Because the previous one is pointing to my fork, the new one is the coverity instead for this repository :)

Comment (eharris) [reply README.md]: Hmm, both seem to point to https://github.com/airsonic/airsonic.git but the one you're changing to doesn't list Javascript as a secondary language so only checks about 40k lines of code instead of nearly 80k lines of code.  The previous one also has more history.

Comment (jvoisin) [reply README.md]: The badge on the [project's webpage]( https://scan.coverity.com/projects/airsonic?tab=overview ) is at https://scan.coverity.com/projects/17971/badge.svg?flat=1

About history and coverage, those are outdated: the project wasn't scanned since April 2019, where I launched it manually, while the previous one is still running on my fork I think.

**Comment by jvoisin** [comment]
@eharris I just relaunched a scan: 251 defects detected, 128 eliminated :)

**Comment by eharris** [comment]
Cool on the reduced defects!  The line count even went down from almost 80k lines to under 77k.  I count that as a win too!

But it looks like you launched a scan on the old project that you are removing the link to (https://scan.coverity.com/projects/17971) vs the new one that doesn't look as complete (https://scan.coverity.com/projects/18356).

**Comment by jvoisin** [comment]
Erf, you're absolutely right, I mixed the projects up :/

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 Airsonic
 ========
 [![Build Status](https://travis-ci.org/airsonic/airsonic.svg?branch=master)](https://travis-ci.org/airsonic/airsonic)
-[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic)
+[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic-airsonic)
```
Comment (eharris) [inline README.md]: I think this particular line doesn't need changing at all.  Now the two links in this line point to different coverity projects.  I believe `17971` (and `.../projects/airsonic`) is the correct one, which is the same as what was originally used.

Comment (jvoisin) [reply README.md]: Sigh, I really need some sleep.
Thanks for being picky ♥

File: .travis.yml [hunk 4]
```diff
@@ -7,6 +7,20 @@ script:
...
   include:
+    - name: coverity
+      script:
+        - echo "coverity scan script"
+      install:
+        - echo "coverity scan install"
+      addons:
+        coverity_scan:
+          project:
+            name: "airsonic/airsonic"
```
Comment (eharris) [inline .travis.yml]: Should this be `airsonic/airsonic` or should it be just `airsonic`?

Comment (eharris) [reply .travis.yml]: Ah, I think it might be this way because it includes the user part of the github project?  In that case nevermind.

Comment (jvoisin) [reply .travis.yml]: It [includes]( https://scan.coverity.com/travis_ci ) the user part :)
